test/setup-beam.test.js

```diff
@@ -104,71 +119,77 @@ async function testOTPVersions() {
   let expected
   let spec
   let osVersion
-  const hexMirrors = ['https://repo.hex.pm', 'https://cdn.jsdelivr.net/hex']
+  const hexMirrors = simulateInput(
+    'hexpm-mirrors',
+    'https://repo.hex.pm, https://cdn.jsdelivr.net/hex',
+    { multiline: true },
+  )
 
   if (process.platform === 'linux') {
     spec = '19.3.x'
     osVersion = 'ubuntu-16.04'
     expected = 'OTP-19.3.6.13'
-    got = await setupBeam.getOTPVersion(spec, osVersion, hexMirrors)
+    got = await setupBeam.getOTPVersion(spec, osVersion)
```

Comment:
I think `hexMirrors` was just exposed for testing purposes, which is not required, since we have a way to simulate inputs.